iUAEParents/MultiPeerConnectivityController.mm

```diff
@@ -193,7 +193,7 @@ - (void)controllerDisconnected:(NSString *)dID {
 
 - (void)setkeymapfordeviceID:(NSString *)dID {
     
-    int kmNumber = [_dMap indexOfObject:dID];
+    NSUInteger kmNumber = [_dMap indexOfObject:dID];
     
```

Comment:
with type int new controller were not be mapped at all. Because int can not store NSNotFound which is long. With this change joystick works now...